src/language/tekster_nn.ts

```diff
@@ -1,681 +1,813 @@
 // tslint:disable:max-line-length
 export default {
-  'skolepenger.sidetittel': 'Søknad om skolepenger',
-  'skolepenger.overskrift': 'Søknad om stønad til skolepenger',
+  // ------ SKOLEPENGER
 
-  'barnetilsyn.sidetittel': 'Søknad om barnetilsyn',
-  'barnetilsyn.tekst.hvilke': 'Hvilke barn skal være med i søknaden?',
+  'skolepenger.sidetittel': 'Application for school fees',
```

Comment:
Virker som att nynorsk fikk engelske tekster

Reply:
gahh! Bra sett, fikser

Reply:
Det er fordi nynorsk ikke ennå har fått sine tekster in general :) 

Men de engelske tekstene har bedre "formattering" på tekstidene, så har bare copy pasta de engelske tekstene over for nå. Planen var å ta litt og litt nynorsk tekster mens jeg venter på noen greier til å kunne legge til engelske tekster på navno

Reply:
![image](https://user-images.githubusercontent.com/402915/98822738-7b908b80-2431-11eb-9ddc-43ee92b275a0.png)
Jeg har kvittet meg med min sånn, men den som har en slik liggende kan vel ta nynorsk-oversettjinga 😂

Reply:
@charliemidtlyng Du angrer det sikker nå!

Reply:
Nei, jeg satt i 15 minutter på eksamen og prøvde å finne ut hvordan "kirke" skrives... Det hjelper ikke med ordbok når du ikke kan ting... Til info så lærte jeg det til slutt; `kyrkje`